apps/frontend/src/shared-research/__tests__/export.test.ts

```diff
@@ -158,20 +176,40 @@ describe('createCsvFileStream', () => {
       a: 'test',
     });
 
-    expect(mockWriteStream.write.mock.calls[0].toString()).toEqual('a,b');
-    expect(mockWriteStream.write.mock.calls[1].toString()).toMatch(
+    expect(mockWriteStream.write.mock.calls[1].toString()).toEqual('a,b');
```

Comment:
I don't like this. The problem is calls[1] so i cant use toHaveBeenCalledWith. I have had to increase this number because the first write is writeDom so the CSV supports UTF-8 characters which are common in description